doc/user-manual/tools/command-line-options.rst

```diff
@@ -427,13 +427,23 @@ Individual warnings can be turned on and off by ``-W {Name}`` and ``-W
 :samp:`UselessPublic`
       ``public`` blocks where they have no effect.
 
-For example, the following command runs Agda with all warnings
+Examples
+~~~~~~~~
+
+Run Agda with all warnings
 enabled, except for warnings about empty ``abstract`` blocks:
 
 .. code-block:: console
 
    agda -W all --warning noEmptyAbstract file.agda
 
+Run Agda on a file which uses the standard library.
+Note that you must have already created a ``libraries`` file
+as described in :ref:`package-system`
+
+.. code-block:: console
+
+   agda -l standard-library -i. file.agda
```

Comment:
Or simply `agda file.agda` if you put the standard library in your defaults file.

Reply:
It looks like a bug about the option `--library=LIB`. Isn't the effect of `--library=LIB` equivalent to adding `LIB` to default? The error message suggests that `file.agda` should be placed in the root directory of the standard library, but this behaviour is not quite right.

I will create an issue if it is really a bug.